src/main/java/org/jabref/gui/preferences/PreviewTabView.java

```diff
@@ -0,0 +1,328 @@
+        );
+        contextMenu.getStyleClass().add("context-menu");
+
+        availableListView.itemsProperty().bind(viewModel.availableListProperty());
+        viewModel.availableSelectionModelProperty().setValue(availableListView.getSelectionModel());
+        new ViewModelListCellFactory<PreviewLayout>()
+                .withText(PreviewLayout::getName)
+                .setOnDragOver(this::dragOverAvailable)
+                .install(availableListView);
+        availableListView.setOnDragDetected(event -> dragDetected(availableListView, event));
+        availableListView.setOnDragDropped(event -> dragDropped(availableListView, event));
+        availableListView.setOnKeyTyped(event -> jumpToSearchKey(availableListView, event));
+        availableListView.getSelectionModel().setSelectionMode(SelectionMode.MULTIPLE);
+
+        chosenListView.itemsProperty().bind(viewModel.chosenListProperty());
+        viewModel.chosenSelectionModelProperty().setValue(chosenListView.getSelectionModel());
+        new ViewModelListCellFactory<PreviewLayout>()
+                .withText(PreviewLayout::getName)
+                .install(chosenListView);
+        chosenListView.setOnDragOver(this::dragOverChosen);
+        chosenListView.setOnDragDetected(event -> dragDetected(chosenListView, event));
+        chosenListView.setOnDragDropped(event -> dragDropped(chosenListView, event));
+        chosenListView.setOnKeyTyped(event -> jumpToSearchKey(chosenListView, event));
+        chosenListView.getSelectionModel().setSelectionMode(SelectionMode.MULTIPLE);
+
+        toRightButton.disableProperty().bind(viewModel.availableSelectionModelProperty().getValue().selectedItemProperty().isNull());
+
+        toLeftButton.disableProperty().bind(viewModel.chosenSelectionModelProperty().getValue().selectedItemProperty().isNull());
+        sortUpButton.disableProperty().bind(viewModel.chosenSelectionModelProperty().getValue().selectedItemProperty().isNull());
+        sortDownButton.disableProperty().bind(viewModel.chosenSelectionModelProperty().getValue().selectedItemProperty().isNull());
+
+        previewPane.setContent(new PreviewViewer(new BibDatabaseContext(), dialogService, Globals.stateManager));
+        // previewPane.setMaxWidth(650); // FixMe: PreviewViewer is too large
+        // previewPane.getContent().maxWidth(650);
+        ((PreviewViewer) previewPane.getContent()).setEntry(TestEntry.getTestEntry());
+        ((PreviewViewer) previewPane.getContent()).setLayout(viewModel.getCurrentLayout());
+        previewPane.visibleProperty().bind(viewModel.chosenSelectionModelProperty().getValue().selectedItemProperty().isNotNull());
+
+        editArea.clear();
+        editArea.setParagraphGraphicFactory(LineNumberFactory.get(editArea));
+        editArea.textProperty().addListener((observable, oldText, newText) -> editArea.setStyleSpans(0, viewModel.computeHighlighting(newText)));
```

Comment:
Have you checked if the parent window has setResizable(true) set?

Reply:
The parent window would be the preferences-dialog, the immediate parent of the PreviewViewer is a TabPane. I believe this is something about the webView the Preview is internally shown in.